package.json

```diff
@@ -40,6 +40,7 @@
     "bcu": "node src/build-browsercheck-utils.js && prettier --write src/browsercheck-utils.js",
     "fa-subset": "node src/fa-subset.js",
     "prepare": "husky",
+    "eslint-inspect": "pnpm dlx pnpm --package=eslint --package=@eslint/config-inspector eslint-config-inspector",
```

Comment:
This thing is actually super cool. Try it out!

Reply:
I want to, but it just gives me:

```
 ERR_PNPM_RECURSIVE_EXEC_FIRST_FAIL  Command "eslint-config-inspector" not found

Did you mean "pnpm eslint-inspect"?
 ELIFECYCLE  Command failed with exit code 1.
```
on Windows :/

Reply:
Hm, I guess I had the command wrong for some reason. I updated it and it should work now.